### ISSUE TITLE
День 2

### ISSUE THREAD

**Issue by muzykaira** [muzykaira/muzykaira.github.io#3]
Харчуємось 3 рази в день: сніданок, обід і вечеря.

Обов’язково! Можна влаштувати перекус, але не канапками, а, наприклад, фруктами, овочами, кефіром або сиром. Сніданок - не пізніше 10 ранку, обід - не пізніше 14, а вечеря - не пізніше 19 години.
Випиваємо не менше 1,5 літра води. Першу склянку кип'яченої води випийте натщесерце. Бажано з лимонним соком ( якщо у вас немає проблем зі шлунком). Відмовляємось від газованої води.
Не їжте чіпси, сухарики та інший фаст-фуд. Те саме стосується й інших забігайлівок з шкідливою їжею.
Замініть свій десерт двома яблуками або іншими фруктами.

**Comment by fitnessblogger** [comment]
Варто уточнити, що кисломолочні продукти не рекомендовано їсти після 17:00.

**Comment by muzykaira** [comment]
@fitnessblogger чи не могла б ти пояснити, на що впливає споживання кисломолочних продуктів у вечірні години доби?

**Comment by fitnessblogger** [comment]
@muzykaira залюбки! Я залишу посилання, де ви зможете знайти всю необхідну інформацію щодо цього питання: https://www.ilady.in.ua/2016/05/koly-jisty-syr.html А якщо по-простому то: наш організм досить довго засвоює білок. Тому, якщо ви полюбляєте кисломолочний сир (або ж інші продукти) більшої жирності, то споживайте їх у першій половині дня.

**Comment by muzykaira** [comment]
@fitnessblogger дуже вдячна за таку інформацію. Впевнена, що вона буде корисною і для інших учасників нашого проекту.

**Comment by fitnessblogger** [comment]
@muzykaira до речі, якщо ми вже заговорили про учасників, то хочеться побажати їх наснаги і терпіння на цьому нелегкому шляху до ідеальної зовнішності.

**Comment by muzykaira** [comment]
@fitnessblogger так, шлях справді довготривалий, але результат однозначно вартий всіх старань.
